src/Models/Entities/Movie.cs

```diff
@@ -21,5 +21,7 @@ public class Movie : BaseEntity
         [Required]
         [Url]
         public string Url { get; set; }
+
+        public List<Favourites> Favorites { get; set; }
```

Comment:
To use "u" or not to use

Reply:
that is the question

Reply:
Boris Johnson > Sleepy Joe
Boris Johnson approves u in Favourite

Reply:
Done